samples/GuestConfiguration/audit-installed-application-linux/README.md

```diff
@@ -0,0 +1,42 @@
+# Audit for installed application - Linux
```

Comment:
>Audit for installed application - Linux [](start = 2, length = 39)

Can you update this to match the policy display name?

Reply:
updated title